Targets/Sample/Sources/02Collection/UIKit/UIKitWorkspaceViewcontrollerRepresenataion.swift

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
Please show `preview` screenshot at creating new view.
It makes this code more easy for understanding.